web/templates/search_description.hbs

```diff
@@ -12,7 +12,7 @@
         {{~!-- squash whitespace --~}}
     {{else if (eq this.type "prefix_for_operator")}}
         {{~!-- squash whitespace --~}}
-        {{this.prefix_for_operator}} {{this.operand}}
+        {{this.prefix_for_operator}} {{this.operand}}{{#if (or (eq this.operand "link") (eq this.operand "image") (eq this.operand "attachment") (eq this.operand "reaction"))}}s{{/if}}
```

Comment:
This would't work with `i18n`.
I think this is why we used the existing wording in the first place - to avoid issues with singular/plural in other languages.

Reply:
Also wouldn't it be cleaner if we made this singular/plural decision in `filter.ts` rather than here?

Reply:
This whole system doesn't support internationalization properly, but certainly it'll make it easier to fix that to have a full phrase constructed in a single place in `filter.ts`.

Reply:
Maybe the best we can do without rewriting the whole system for generating these strings is what you have here. But we should definitely redo the whole way these strings are constructed for fix i18n for search suggestions as a follow-up, not for 9.0. 

Ultimately the whole system of trying to glue together operands like this is unworkable, we should just have JS code that for each of the ~25 types generates the full string desired without variable substitution on operators/operands at all except where they reflect user data, like a user.

Reply:
> I think manipulating `operand` in `search_terms` will just complicate it unless we change how we render `{{this.prefix_for_operator}} {{this.operand}}` completely.

I think we should just map suggestion strings for each operators/operands and render them just like we do in search suggestion items which would make these type of changes easier to work on.